types.go

```diff
@@ -0,0 +1,11 @@
+package cast
+
+type (
+	Float64er interface {
```

Comment:
No need to export these (as then they would have to have some kind of documentation)